docs/developer-guide/viewports.md

```diff
@@ -1,47 +0,0 @@
-# Viewports
```

Comment:
Isn't this kinda useful, even as a non-published doc for maintainers?

Reply:
Some of the content is already covered by the `Views` article; some parts are outdated or incomplete. Even if the page is not listed in the table of contents, it will show up in search. I prefer not to publish misleading information.